gamemodes/terrortown/gamemode/shared/sh_entity.lua

```diff
@@ -0,0 +1,18 @@
+---
+-- @ref https://wiki.facepunch.com/gmod/Entity
+-- @class Entity
+
+local entmeta = FindMetaTable("Entity")
+if not entmeta then
+    return
+end
+
+---
+-- Checks if the entity is a ragdoll. It also makes sure it is a true ragdoll
+-- by checking if it has a player nick name because ragdolls can also be used
+-- for props (e.g. a mattress).
+-- @return boolean Returns true if it is a true ragdoll
+-- @realm shared
+function entmeta:IsTrueRagdoll()
```

Comment:
Wouldn't something like `IsPlayerRagdoll` make more sense? 

"True" can be subjet to interpretation :)

Reply:
I wasn't happy with my naming either, your suggestion is quite nice, thanks!